src/CoreBundle/Event/HtmlIncludeEvent.php

```diff
@@ -68,7 +68,7 @@ private function isInteger($value)
     {
         if (true === is_numeric($value)) {
             $test = (int) $value;
-            if (0 !== $test || '0' == $value) {
```

Comment:
This check should be re-examined. It seemed nonsensical while implementing it, but I am unsure, looking at it now in github